### ISSUE TITLE
[FEATURE REQUEST] Add overview about notes+

### ISSUE THREAD

**Issue by HarriBuh** [NimaKhajehpour/MyMood#14]
It would be nice to have an overview in a calendar-like design where you can have an easy and quick look over your saved entries.

Another request from me is to add a time-stamp to your entries to make the ability to save several entries per day actually functional.
Like "12:00 😀 Had lunch with xy"; "16:00 😐 Sat on my couch, bored."

**Comment by NimaKhajehpour** [comment]
Hey there, thank you for using my app and your suggestions ❤️. For the calendar like overview, i saw it on other mood tracking apps and i thought it would be good to add it and now that you have mentioned it, i have one more reason to do that 😀 as for the second request can you provide more context on what exactly you want to have? Because you can add more than one effect entry for every day so i assume that i did not fully understand what you mean.

**Comment by HarriBuh** [comment]
Thank you for your fast and kind answer.

What I mean is that you can already add several entries per day, BUT you don't know what time you wrote them- which would be quite important for "moody" people like me to get a better understanding of it.

I'll try another, formatted example:

11:00: 😀 Breakfast with family
17:00: 😐 Got bored
23:00: ☹️ Couldn't get to sleep

The only difference to the current design would be to add the time of every entry in front of the text.

I hope this helped you more, have a great day.

**Comment by NimaKhajehpour** [comment]
Yup i get it now 😀 i guess you described it pretty well in the first place. I just did not get it. So it already has date on it and for the time it does make sense to add a timestamp too. You are right that is what a moody app would absolutely need 😁i will have big changes for the next update and i won't forget about these features. Thank you again for using my app and helping it improve and ge better ❤️

**Comment by HarriBuh** [comment]
A shame the enhancement didn't make it into the latest update. I fact, I don't see any significant or "big" improvements.. 🫢 It's still on your radar anyway, right?

**Comment by NimaKhajehpour** [comment]
yeah the latest build i made was a maintenance build to test something on fdroid. as you can see on issues list the latest build on fdroid has failed the RB (reproducible build) which enables user to download the app from wherever they want without any problems. so yes all the updates and requests are still planned and will be implemented. on the other hand my work is getting slower until the problem is either fixed or at least we know the reason of the fails. thanks for your concern and thank you for your patience. :heart:

**Comment by NimaKhajehpour** [comment]
@HarriBuh hey did you change your mind on this request?

**Comment by NimaKhajehpour** [comment]
Hey no offense taken. Im sorry for the inconvenience of delays on app updates. I got into sone work and found a job recently so it took a little too much time. I am still going to add the feature and will try to maintain the app sooner. But with the amiunt of used apps on fdroid i have and the job it might take a little longer. Again i am sorry that this issue was ipen for so long. Feel free to open it again whenever you want knowing the fact that i will implement it.

**Comment by NimaKhajehpour** [comment]
Hey @HarriBuh, remember the big change i talked about and it did not happen for a long time?
just thought i would tell you sooner that the big change is coming. well not as big of a change. its just some ui enhancements, theme and overviews and guess what? the calendar view is going to make it in this update.
im still on R&D stage of the feature but the app with the requested feature will be out in 2 or 3 days.
thanks again for contributing, opening this issue and requesting the feature ❤️

**Comment by NimaKhajehpour** [comment]
im gonna open this issue and properly close it this time if you dont mind.

**Comment by NimaKhajehpour** [comment]
yup and that requires a custom view which I am working on. i pushed it with a simple calendar overview so that we have a base of what we want and i will add the custom made view on next update

**Comment by NimaKhajehpour** [comment]
hey @HarriBuh i know we really wanted the calendar view and i am very sorry that i have removed even the small calendar view that we had as well.
but i thought now that I am making a big change that I talked about, i remove the half worked calendar view and add it on the next update with the proper implementation.
sorry for taking so long but i am working on it and will hopefully have most of the issues as well as this one properly completed.
thanks for using my app and also being patient.

**Comment by NimaKhajehpour** [comment]
with all these being said i re-opened this issue to properly close it this time with implementing the feature as we would actually like it to be.

**Comment by HarriBuh** [comment]
It's been over 3 months now, any news? I don't like abandoned tickets, like I already said back in 2023.

**Comment by HarriBuh** [comment]
Is this still valid ? Just wondering because this ticket has been left open for years. Could just close it as well.
